.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: tests
+on:
+  push:
+    branches:
+      - master
+      - dev
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - dev
+      - master
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 40 # default is 360
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm install
+      - name: Test
+        run: npm run test
```

Comment:
What about the coverage report? It would be great to upload it somewhere.

Reply:
Coverage report uploaded to [codecov](https://codecov.io/gh/HathorNetwork/hathor-wallet-headless/tree/744fc3a14c09fd57e56d35a44f49f37dee41c846/src).